assets/section-image-banner.css

```diff
@@ -436,3 +436,19 @@
 .banner__box > * + .banner__buttons {
   margin-top: 2rem;
 }
+
+@media screen and (max-width: 749px) {
+  .banner:not(.slideshow) .rte a,
+  .banner:not(.slideshow) .inline-richtext a:hover,
+  .banner:not(.slideshow) .rte a:hover {
+    color: currentColor;
+  }
+}
```

Comment:
It doesn't _technically_ need to. But in this case, we have multiple rules that are inherited correctly on desktop by default. 

We _could_ definitely remove the `max-width` media query here, but then we'd need to take corrective action on 750 and above to replicate the way it would've normally appeared. This doubled the CSS rules, and seemed heavy-handed to me (especially considering there are lots of other `(max-width: 749px)` breakpoints used in this file already). 

Basically, it's a choice between this: 

```
@media screen and (max-width: 749px) {
  .banner:not(.slideshow) .rte a,
  .banner:not(.slideshow) .inline-richtext a:hover,
  .banner:not(.slideshow) .rte a:hover {
    color: currentColor;
  }
}
```

... or this: 

```
.banner:not(.slideshow) .rte a,
.banner:not(.slideshow) .inline-richtext a:hover,
.banner:not(.slideshow) .rte a:hover {
  color: currentColor;
}

@media screen and (min-width: 750px) {
  .banner:not(.slideshow) .rte a,
  .banner:not(.slideshow) .rte a:hover {
    color: rgba(var(--color-link),var(--alpha-link));
  }

  .banner:not(.slideshow) .inline-richtext a:hover {
    color: var(--color-link);
  }
```

`inherit` doesn't work in this case, so you're stuck specifying the same variables that are assigned in the original rules from [`component-rte.css`](https://github.com/Shopify/dawn/blob/00292c97d067d1d160b4ac2e87d74d945fed26b9/assets/component-rte.css#L50) and [`base.css`](https://github.com/Shopify/dawn/blob/00292c97d067d1d160b4ac2e87d74d945fed26b9/assets/base.css#L753). All in all, I felt that just using the `max-width` rule was simpler. Happy to change if you feel strongly though.